File: src/main/java/br/com/eteg/opbp/repository/AccountRepository.java
package br.com.eteg.opbp.repository;

import org.springframework.data.repository.CrudRepository;
import org.springframework.data.repository.query.Param;
import org.springframework.web.bind.annotation.CrossOrigin;

import br.com.eteg.opbp.entities.Account;

@CrossOrigin
public interface AccountRepository extends CrudRepository<Account, Integer> {
  Account findByUsername(@Param("username") String username);
}

File: src/main/kotlin/br/com/eteg/opbp/repository/AccountRepository.kt
package br.com.eteg.opbp.repository;

import br.com.eteg.opbp.entities.Account
import org.springframework.data.repository.CrudRepository
import org.springframework.data.repository.query.Param
import org.springframework.web.bind.annotation.CrossOrigin

@CrossOrigin
interface AccountRepository : CrudRepository<Account, Int> {
    fun findByUsername(@Param("username") username: String?): Account?
}
